crates/service-protocol/build.rs

```diff
@@ -18,8 +18,12 @@ fn main() -> std::io::Result<()> {
     prost_build::Config::new()
         .bytes(["."])
         .protoc_arg("--experimental_allow_proto3_optional")
+        .enum_attribute("protocol.Version", "#[derive(::strum_macros::FromRepr)]")
         .compile_protos(
-            &["service-protocol/dev/restate/service/protocol.proto"],
+            &[
+                "service-protocol/dev/restate/service/protocol.proto",
+                "service-protocol/dev/restate/service/discovery.proto",
```

Comment:
The name of this proto file is :shrug: because it makes me think the discovery manifest is there.

Reply:
It contains the protobuf related discovery types. If you have a better name in mind then let me know.